src/pages/accounts/ForgotPassword.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import Logo from 'molecules/Logo';
+import ActionButton from 'components/ActionButton';
+import TextButton from 'components/TextButton';
+import TextInput from 'components/TextInput';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+import { Auth } from 'aws-amplify';
+
+const ForgotPasswordPageContainer = styled.div`
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	background-color: ${({ theme }) => theme.colors.main[1]};
+`;
+
+const ForgotPasswordContents = styled.div`
+	max-width: 300px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: center;
+	grid-gap: 1em;
+`;
+
+const PassrLogo = styled(Logo)`
+	width: 100%;
+`;
+
+const SubmitButton = styled(ActionButton)`
+	width: 100%;
+	padding: 1em;
+`;
+
+const TextLinkContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	width: 100%;
+`;
+
+const TextLink = styled(TextButton)`
+	font-size: ${({ theme }) => theme.fontSizes.s};
+	font-weight: 500;
+	margin: 0.5em;
+	word-wrap: break-word;
+`;
+
+function validEmail(email: string) {
+	return !!email.match(/.+@.+\..{2,}/);
+}
+
+function ForgotPasswordPage() {
+	const [email, setEmail] = useState('');
+	const [emailErr, setEmailErr] = useState(false);
+
+	const history = useHistory();
+
+	useEffect(() => {
+		Auth.currentAuthenticatedUser()
+			.then(() => {
+				history.push('/');
+			})
+			.catch(() => {
+				console.log('User not signed in.');
+			});
+	}, []);
+
+	useEffect(() => {
+		const listener = (event: KeyboardEvent) => {
+			if (event.code === 'Enter' || event.code === 'NumpadEnter') {
+				console.log('Enter pressed!');
+				handleSubmit();
+			}
+		};
+		document.addEventListener('keydown', listener);
+		return () => {
+			document.removeEventListener('keydown', listener);
+		};
+	}, [email]);
+
+	const handleSubmit = async () => {
+		if (!validEmail(email)) {
+			setEmailErr(true);
+			return;
+		}
+		console.log(email);
+	};
+
+	return (
+		<ForgotPasswordPageContainer>
+			<ForgotPasswordContents>
+				<PassrLogo width="300px" height="300px" />
+				<TextInput
+					value={email}
+					onChange={(e) => {
+						if (validEmail(e.target.value)) setEmailErr(false);
+						setEmail(e.target.value);
+					}}
+					onBlur={(e) => {
+						if (!validEmail(e.target.value)) setEmailErr(true);
+					}}
+					label="Email"
+					required
+					error={emailErr}
+					placeholder="Email"
+				/>
+				<SubmitButton
+					variant="primary"
+					disabled={emailErr}
+					onClick={handleSubmit}
+				>
+					Submit
+				</SubmitButton>
+				<TextLinkContainer>
```

Comment:
right above this line I removed a line that was {userPassError && <h1>Could not sign in.</h1>} does it make sense to remove this since the only check we need for entering information into this page is the validEmail check?

Reply:
Yeah, I would remove it. We may add something to depict an error, but it depends on which errors we would get back from submitting the email to AWS. TL;DR remove it. We will add something later if necessary.